### ISSUE TITLE
Colors are not shown on Razer devices

### ISSUE THREAD

**Issue by Yunoo** [bengreenier/overlights#13]
Colors that are send from Overwatch to Razer Chroma SDK / Razer Chroma Client are not shown on any Razer devices after Overlights installation. The only place where the colors are actually shown is only Overlights UI. Are there any possibility for the color events to be consumed both by this app and Razer Choma devices at the same time? 

Could it be because of the compatibility issues with different DLL and Razer Chroma SDK (Razer Client, Razer Synapse etc) versions? If so, then could you provide the version of the SDK / Synapse / DLL that you used for creating your RzChromaSDK64.dll? Thanks.

**Comment by bengreenier** [comment]
This is correct. Since we directly patch the dlls we lose the ability to page out to razer devices. For now, this is not a goal of this project - that said, it's definitely possible to also call out to the original razer dlls to send data to the keyboard as well, and i'd happily take a PR there.
